packages/mcfunction/src/colorizer/macro.ts

```diff
@@ -0,0 +1,39 @@
+import * as core from '@spyglassmc/core'
+import type { MacroNode } from '../node/macro'
+
+export const macro: core.Colorizer<MacroNode> = (node, ctx) => {
+	const tokens: core.ColorToken[] = []
+	tokens.push(
+		core.ColorToken.create(
+			core.Range.create(node.range.start, node.range.start + 1),
+			'literal',
+		),
+	) // Dollar Sign
+	for (const child of node.children) {
+		if (child.type === 'mcfunction:macro/other') {
+			tokens.push(core.ColorToken.create(child.range, 'string')) // Other stuff is Orange
```

Comment:
I don't know how I feel about describing semantic color tokens with names of colors (same with the added comments in `packages/core/src/processor/colorizer/Colorizer.ts`). It is not the language server's concern what color this text should be and it is beyond the language server's capabilities to set those colors. The language server should choose a token based on purely the semantic meaning of the text and select the existing token with the closest alignment (in this case, `string` is a good choice due to macro line's similarity to template strings in other programming languages).

Reply:
So you want me to remove the comments in Colorizer as well?